### PR TITLE
fix scrolling

### LINKED ISSUE #3793: scroll bar should be on edge of screen, not within margin
### Observed behavior

regression caused by #3782 - scroll bar has margin to its right:

screen shot 2018-06-13 at 11 09 01 am

reported here: https://github.com/learningequality/kolibri/pull/3778#issuecomment-396801750

### Expected behavior

scroll bar should be on page edge

### User-facing consequences

layout looks broken

### Context

0.10.0-beta2

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

fixes #3793

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Review by christianmemije — CHANGES_REQUESTED** [review]
The error-box needs to also have this max-width, so instead wrap the entire template.

```html
<div class="wrapper" v-else>
  <error-box v-if="error" />
  <slot></slot>
</div>
```

**Comment by indirectlylit** [comment]
good catch. pushed update

**Review by christianmemije — APPROVED** [review]
Acceptable.

File: kolibri/core/assets/src/views/app-body.vue [hunk 1]
```diff
@@ -10,8 +10,10 @@
       :delay="false"
     />
     <template v-else>
```
Comment (christianmemije) [inline kolibri/core/assets/src/views/app-body.vue:12]: You could have replaced the template with the div.

Comment (indirectlylit) [reply kolibri/core/assets/src/views/app-body.vue:12]: good catch - PR here #3817

Comment (indirectlylit) [reply kolibri/core/assets/src/views/app-body.vue:12]: (I see now that's what you wrote above originally)

**Pull request merged**